AUTHORS

```diff
@@ -79,28 +94,41 @@ Andy Rothfusz <github@developersupport.net>
 Andy Smith <github@anarkystic.com>
 Andy Wilson <wilson.andrew.j+github@gmail.com>
 Anes Hasicic <anes.hasicic@gmail.com>
+angt <angt@users.noreply.github.com>
+Anil Belur <askb23@gmail.com>
 Ankush Agarwal <ankushagarwal11@gmail.com>
 Anthony Baire <Anthony.Baire@irisa.fr>
 Anthony Bishopric <git@anthonybishopric.com>
+Anthony Dahanne <anthony.dahanne@gmail.com>
 Anton Löfgren <anton.lofgren@gmail.com>
 Anton Nikitin <anton.k.nikitin@gmail.com>
+Anton Polonskiy <anton.polonskiy@gmail.com>
 Anton Tiurin <noxiouz@yandex.ru>
+Antonio Murdaca <amurdaca@redhat.com>
 Antonio Murdaca <antonio.murdaca@gmail.com>
+Antonio Murdaca <runcom@redhat.com>
```

Comment:
This is a better example because it's pretty clear these are probably all @runcom :smile: